### PR TITLE
Update SAML tutorial with new default Admin settings

### PR DESCRIPTION
**Pull request by Naarcha-AWS**
This PR updates the SAML tutorial based off of PR #5887 

### Checklist
- [x] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _security/authentication-backends/saml.md [hunk 1]
```diff
@@ -34,20 +34,20 @@ We provide a fully functional example that can help you understand how to use SA
    
    ```yml
    opensearch-saml-node1:
-    image: opensearchproject/opensearch:2.8.0
+    image: opensearchproject/opensearch:2.12.0
```
Comment (DarshitChanpura) [inline _security/authentication-backends/saml.md:37]: we should change version on line 33 as well since it points to 2.6

**Review by DarshitChanpura — COMMENTED** [review]
Ty @Naarcha-AWS for taking care of this!!

**Comment by Naarcha-AWS** [comment]
@DarshitChanpura: I went ahead and used our site variable here so it'll update to whatever version the user is on currently.

**Review by DarshitChanpura — APPROVED** [review]
Nicely done @Naarcha-AWS !!

File: _security/authentication-backends/saml.md [hunk 2]
```diff
@@ -30,24 +30,24 @@ We provide a fully functional example that can help you understand how to use SA
    You can remove "customize" from the file names if you plan to modify and keep these files for production.
    {: .tip }  
 
-1. In the `docker-compose.yml` file, specify your OpenSearch version number in the `image` field for nodes 1 and 2, and OpenSearch Dashboards server. For example, if you are running OpenSearch version 2.6, the `image` fields will resemble the following examples:
+1. In the `docker-compose.yml` file, specify your OpenSearch version number in the `image` field for nodes 1 and 2, and OpenSearch Dashboards server. For example, if you are running OpenSearch version {{site.opensearch_major_minor_version}}, the `image` fields will resemble the following examples:
```
[the remark this answers is not in the document]
Comment (vagimeli) [reply _security/authentication-backends/saml.md:33]: Should {{site.opensearch_major_minor_version}} be in code font?

**Pull request merged**